datalad/plugin/wtf.py

```diff
@@ -104,16 +104,27 @@ def _describe_annex():
         info[key] = value
     return info
 
-
 def _describe_system():
     import platform as pl
     from datalad import get_encoding_info
+
+    if hasattr(pl, 'dist'):
+        dist = pl.dist()
+    else:
+        # Python 3.8 removed .dist but recommended "distro" is slow, so we
+        # try it only if needed
+        try:
+            import distro
+            dist = distro.linux_distribution(full_distribution_name=False)
+        except Exception as exc:
```

Comment:
nitpick 1: I'd prefer to narrow the catch to an import error.
nitpick 2: No need to bind `exc`.

Also, should we log a warning here?

Reply:
I smile when I see some diagnostic command to provide information about a problem blows itself up due to some other issue...  So I would prefer to capture `Exception` here as well, since I do not trust (yet) `distro` to not blow.  So now doing dedicated captures... 
I again pushed before getting rev-diff quickly, sorry... I will get there

Reply:
What you went with looks fine to me.  Thanks.